imageresizerapp/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,5 @@
+Uninstall-ChocolateyPackage `
+  -PackageName 'imageresizerapp' `
+  -FileType 'EXE' `
+  -Silent '/quiet /norestart /uninstall' `
```

Comment:
It's odd that `/uninstall` is listed here. It should have a more formal place than inside `-Silent`.